spec/facets/CatalogDatasetFacet.json

```diff
@@ -0,0 +1,51 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://openlineage.io/spec/facets/1-0-0/CatalogDatasetFacet.json",
+  "$defs": {
+    "CatalogDatasetFacet": {
+      "allOf": [
+        {
+          "$ref": "https://openlineage.io/spec/2-0-2/OpenLineage.json#/$defs/DatasetFacet"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "technology": {
+              "description": "The storage framework for which the catalog is configured",
+              "type": "string",
+              "example": "iceberg, delta, hive"
+            },
+            "source": {
```

Comment:
Well, OpenLineage events already have processing engine facet, what's the difference?